docs/pbc/all/tax-management/202311.0/base-shop/third-party-integrations/vertex/install-vertex.md

```diff
@@ -28,674 +28,10 @@ Ensure that your installation meets these requirements.
-| expenses.shipment.shipmentAddress                                                            | lineItems[].customer.destination                                         |                                                                                                                                                 |
-| billingAddress                                                                               | lineItems[].customer.administrativeDestination                           |                                                                                                                                                 |
-| expenses.sumPrice (either GROSS or NET depending on currently selected mode)                 | lineItems[].extendedPrice                                                |                                                                                                                                                 |
-| expenses.sumDiscountAmountAggregation                                                        | lineItems[].discount.discountValue                                       | Prices are converted from the Spryker’s cent-based format to the Vertex decimal format.                                                                  |
-| -                                                                                            | lineItems[].discount.discountType                                        | Always `DiscountAmount`. Spryker stores discount based on amount, so there is no need to use percentage-based discounts.                   |
-
-##### Location mapping
-
-| Spryker          | Vertex         | Comment                            |
-|------------------|----------------|------------------------------------|
-| address1         | streetAddress1 |                                    |
-| address2         | streetAddress2 | Should be either not empty or null |
-| city             | city           |                                    |
-| state            | mainDivision   | Should be either not empty or null |
-| zipCode          | postalCode     |                                    |
-| country.iso2Code | country        |                                    |
+1. [Integrate ACP connector module for tax calculation](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/integrate-connector-module.html)
+2. [Integrate Vertex app](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/integrate-vertex-app.html)
 
 ## Next step
-[Configure Vertex in the Back Office](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/configure-vertex.html)
+[Configure Vertex in the Back Office](/docs/pbc/all/tax-management/{{page.version}}/base-shop/third-party-integrations/vertex/configure-vertex.html)
```

Comment:
I've kept this "next step" section here as it is a next step after installation is finished; at the same time the same section is present in "integrate-vertex-app.md" file. Should we keep it like this or should it be present in both files?

Also: should we keep the folder structure like this or should new files be moved lower in the hierarchy?